oracle/block.go

```diff
@@ -279,6 +280,28 @@ func (b *FullBlock) SetEvents(events *Events) {
 	}
 
 	b.Events = events
+
+	// Special case. Temporal fix. We artificially add the mev reward for this block.
+	// See: https://github.com/dappnode/mev-sp-oracle/pull/230
+	// See: https://beaconcha.in/slot/10400574
+	if b.ChainId == MainnetChainId && b.GetSlotUint64() == ExceptionSlotMainnet1 {
+		log.Warn("Special case: MEV reward hardcoded. See MevRewardInWei for more info")
+		b.Events.EtherReceived = append(b.Events.EtherReceived, &contract.ContractEtherReceived{
+			Sender:         common.Address{},
+			DonationAmount: big.NewInt(177043568463114308),
+			Raw: types.Log{
```

Comment:
are we sure is it okay to let as undefined or `0` so many fields of this event? at least `BlockNumber` should be easy to define

Reply:
good point, but prefer to leave it as 0/{} to be very explicit that its a fake event.